lib/litle.rb

```diff
@@ -27,6 +27,14 @@
 module ActiveMerchant #:nodoc:
   module Billing #:nodoc:
     class LitleGateway < Gateway
+      def initialize(options={})
+        if options.has_key?(:test_url)
```

Comment:
Instead of forcing some defaults, can't we simply change this to `unless options[:test_url].blank?`?